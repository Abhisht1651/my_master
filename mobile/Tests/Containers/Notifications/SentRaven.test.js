// @flow
import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import SentRaven from '../../../App/Components/Notifications/SentRaven'

describe('SentRaven', () => {
  it('SentRaven component renders correctly', () => {
    const tree = shallow(<SentRaven />)
    expect(tree).toMatchSnapshot()
  })

  it('should render the Text Component', () => {
    const wrapper = shallow(<SentRaven />)
    expect(wrapper.find('Text')).toHaveLength(3)
  })

  it('should render the Image Component', () => {
    const wrapper = shallow(<SentRaven />)
    expect(wrapper.find('Image')).toHaveLength(2)
  })

  it('should render the Image Component', () => {
    const wrapper = shallow(<SentRaven />)
    expect(wrapper.find('ImageBackground')).toHaveLength(1)
  })
})
