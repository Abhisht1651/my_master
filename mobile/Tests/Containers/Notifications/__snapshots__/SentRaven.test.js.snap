// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SentRaven SentRaven component renders correctly 1`] = `
<View
  style={
    Object {
      "zIndex": 7,
    }
  }
>
  <View
    style={Object {}}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          Object {
            "width": 132,
          }
        }
      >
        <View
          style={
            Object {
              "paddingLeft": 50,
              "paddingTop": 46,
            }
          }
        >
          <ImageBackground
            imageStyle={
              Object {
                "borderRadius": 22.17857142857143,
              }
            }
            source={
              Object {
                "default": "",
              }
            }
            style={
              Object {
                "height": 44.35714285714286,
                "width": 44.35714285714286,
              }
            }
          >
            <Image
              source={
                Object {
                  "default": "",
                }
              }
              style={
                Object {
                  "height": 31.21428571428571,
                  "left": 24.642857142857146,
                  "top": 24.642857142857146,
                  "width": 31.21428571428571,
                }
              }
            />
          </ImageBackground>
        </View>
      </View>
      <View
        style={
          Object {
            "width": 574,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View
            style={
              Object {
                "paddingTop": 42.714285714285715,
                "width": 464,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#8F8FAF",
                  "fontFamily": "BananaGrotesk-Light",
                  "fontSize": 23,
                  "lineHeight": 31.21428571428571,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#1F1F60",
                    "fontFamily": "BananaGrotesk-Semibold",
                  }
                }
              >
                Mia Lewis
              </Text>
               sent you a raven
            </Text>
          </View>
          <View
            style={
              Object {
                "paddingRight": 28.000000000000004,
                "paddingTop": 41.07142857142857,
              }
            }
          >
            <ForwardRef
              activeOpacity={1}
              onPress={[Function]}
            >
              <Image
                source={
                  Object {
                    "default": "",
                  }
                }
                style={
                  Object {
                    "height": 4.928571428571428,
                    "width": 30,
                  }
                }
              />
            </ForwardRef>
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View
            style={
              Object {
                "width": 464,
              }
            }
          />
          <View
            style={
              Object {
                "justifyContent": "flex-end",
                "paddingRight": 28.000000000000004,
                "paddingTop": 4.928571428571428,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#8F8FAF",
                  "fontFamily": "BananaGrotesk-Light",
                  "fontSize": 19.71428571428571,
                  "lineHeight": 23,
                }
              }
            >
              1 day
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingTop": 34.5,
        }
      }
    >
      <View
        style={
          Object {
            "paddingLeft": 53.99999999999999,
            "paddingRight": 46,
          }
        }
      >
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
